src/app/Library/CrudPanel/Traits/Create.php

```diff
@@ -327,7 +327,7 @@ private function createManyEntries($entry, $relation, $relationMethod, $relation
 
         foreach ($items as $item) {
             if (isset($item[$relation_local_key]) && ! empty($item[$relation_local_key])) {
-                $entry->{$relationMethod}()->updateOrCreate([$relation_local_key => $item[$relation_local_key]], Arr::except($item, $relation_local_key));
+                $entry->{$relationMethod}()->where($relation_local_key, $item[$relation_local_key])->update(Arr::except($item, $relation_local_key));
```

Comment:
Why is this change needed? I'm a bit weary because I see `updateOrCreate()` has turned into `update()`.

Reply:
This is the correct, it would never be created here because if `!empty($item[$relation_local_key])` means we have an entry to update, so ... it gets created in the `else` condition.